src/main/java/de/retest/recheck/ui/descriptors/StringAttribute.java

```diff
@@ -44,10 +44,10 @@ public Integer parse( final String value ) throws ParameterParseException {
 	public static final ParameterType parameterTypeBoolean = new ParameterType( "BOOLEAN" ) {
 		@Override
 		public Boolean parse( final String value ) throws ParameterParseException {
-			if ( value.equalsIgnoreCase( "true" ) ) {
```

Comment:
Was genau ist hier der Unterschied, wieso können so NPEs verhindert werden?

Reply:
If `value` is `null` the old code throws a NPE, while `"true".equalsIgnoreCase()` is just `false`.